### PR TITLE
Add session "Streamline your app clip"

### PR DESCRIPTION
**Pull request by rogerluan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -551,4 +552,48 @@ _Jonathan Hersh, Allen Langmaier_
...
+  - Focus on essential tasks
+  - Limit the features. Reserve complex features for your app
+  - Require all assets for initial experience
+  - Do not include splash screens
+  - Avoid requiring people to sign up _before_ finishing their task
+  - Only ask for location and notification permissions if needed (more on this below)
+  - Make sure your App Clip has the same name and icon as your app
+  - Put shared assets in a shared asset catalog
+  - Authentication
+    - Support sign in with apple
```
Comment (Blackjacx) [inline README.md]: Never thought I will ever corect a typo for you 😅 But here it is:

**Sign in with Apple**

File: README.md [hunk 2]
```diff
@@ -551,4 +552,48 @@ _Jonathan Hersh, Allen Langmaier_
...
+      - Check the authorization status in the `NotificationSettings`, under the new `.ephemeral` status
+    - Users can explicitly disable these permissions via App Clip settings panel
+    - App Clips can prompt the user for full Location and Notifications access, although discouraged
+  - Strongly suggest using ApplePay
+  - Strongly suggest using "Sign In With Apple"
+  - Use `AuthenticationServices` API to sign your users in without showing a login screen (assuming they have already previously signed up in your website for instance)
+- **Transition users to your app**
+  - iOS suggests downloading your app from App Clip experiences, banners and App Clip settings
+  - You can embed `StoreKit` `SKOverlay` in a view
+    - Suggestion: display this overlay only after the user finishes their task. E.g. after payment
```
Comment (Blackjacx) [inline README.md]:
```suggestion
    - Suggestion: display this overlay only after the user finishes their task, e.g. after payment
```

**Review by Blackjacx — CHANGES_REQUESTED** [review]
Just some minor typos... Otherwise very good and informative - Thanks 🙏

File: README.md (hunk 1 above)
Comment (rogerluan) [reply README.md]: hahaha 😆 good catch!

Comment (rogerluan) [reply README.md]:
```suggestion
    - Support Sign in with Apple
```

**Review by Blackjacx — APPROVED** [review]

**Pull request merged**